packages/espressocash_client/lib/src/espressocash_client.dart

```diff
@@ -122,7 +129,15 @@ class EspressoCashClient {
     return response.token;
   }
 
+  void _ensureLoggedIn() {
```

Comment:
I don't think it's needed. Clients should check for unauthenticated error response and handle it correctly. They have to do it anyway, because having saved token is not a guarantee that it's still valid on the backend.

Reply:
makes sense. refactored and update client